frontend/apps/filemanager/filemanagerhistory.lua

```diff
@@ -105,11 +104,17 @@ function FileManagerHistory:onMenuHold(item)
         {},
         {
             {
-                text = _("Clear history of deleted files"),
+                text = _("Clean history of deleted items"),
```

Comment:
I don't think the rephrasing is clearly better or something, so I'd prefer if you just left it. 

Put another way, think of the translators. ;-)

Reply:
https://github.com/koreader/koreader/pull/7635#discussion_r627114597

<kbd>![07](https://user-images.githubusercontent.com/62179190/126475649-c820ab27-2ecf-4936-a30a-a5dd96be590f.png)</kbd>
<kbd>![08](https://user-images.githubusercontent.com/62179190/126475650-37d4ead2-7f2a-4c8b-a3b1-712bd60a58c0.png)</kbd>
https://github.com/koreader/koreader/blob/dc59391632219e71a756a6123952d8e1f1d0eea6/frontend/apps/reader/modules/readerdictionary.lua#L296
https://github.com/koreader/koreader/blob/dc59391632219e71a756a6123952d8e1f1d0eea6/frontend/apps/reader/modules/readerwikipedia.lua#L308
https://github.com/koreader/koreader/blob/dc59391632219e71a756a6123952d8e1f1d0eea6/plugins/texteditor.koplugin/main.lua#L202

Reply:
Dunno about `clear` vs `clean`, but I think `deleted files` is better than `deleted items` (it's really _files you have deleted in the past_ - and not _history items that you did delete_ because there are obviously still here if you need that action :) - or it's the file underneath the history item)

This suggested new string is also in #8000 - so we should make them similar in both PRs.

Reply:
Items -> files also here: `Auto-remove deleted or purged items from history`?

Reply:
This one a bit ambiguous: we delete files, but I'm not sure we can say we purge a file, rather that we purge the item (file+ docsettings = item, we purge a part of it).
So, dunno :/

Reply:
<kbd>![1](https://user-images.githubusercontent.com/62179190/126781156-7df02fd0-232c-4c47-a0ef-8058505e6f02.png)</kbd>

Reply:
<kbd>![2](https://user-images.githubusercontent.com/62179190/126781541-80e947bb-fa65-42bb-8ddf-4735dbfd74ab.png)</kbd>

Reply:
Clean means to get rid of, clear means to tidy up. You could say both here because of the qualification, but clear already means the intended meaning all by itself while clean would mean wipe.

Which is to say, `Clear history of deleted files` properly conveys what's going on, which is completely different from where clean is used.

Reply:
Also files may be items, but items aren't files. Being more generic is not better.

Reply:
So, as on these screenshots, with "Clear" instead of "Clean", should be fine :)